Add support form crm onpremise 8.2

bpedziwiatr opened the pull request:
add support for setup teimeout on navigation methods
use default page if exist instead of creating new one
add support for onpremise CRM

DigitalFlow commented:
Hi @bpedziwiatr,

Thanks a lot for contributing!
Your contribution looks fine to me, we just need to adapt the documentation for the changes.
Is there anything that you think would need a special documentation?

Kind Regards,
Florian

PS: The current release would be numbered 0.40 instead of 0.41 as per your PR, but I can fix this after merging.

bpedziwiatr commented:
Hi @DigitalFlow ,
No problem it didn’t work for my on-premise  environment so I added new functionality, without removing existing code for current online version.
Let's  say I will be testing current solution in around 1-2 weeks,  during this time I might do some additional changes. So lets wait with merge until I check everything on real examples.
With documentation I can try to add new options that I added to code.
Ok I will change version to 0.40 instead of 0.41
